docs/dictionary/function/machine.lcdoc

```diff
@@ -30,22 +31,24 @@ The <machine> <function> <return|returns> a <string>.
 -   iPhone Simulator: the device is a simulated iPhone
 -   iPad: the device is the iPad
 -   iPad Simulator: the device is a simulator iPad
-
+-   : macOS, Unix and Android returns described below
```

Comment:
@seaniepie I think we should drop the idea of this being an `enum` and either move the description of the string value to one place or the other rather than having some listed here and some info listed in the description. One paragraph per platform in the description seems like a good idea.

Other info is on windows `the machine` returns the same value as `the processor`. In FileMaker 64 bit and at some point in the future IDE/standalone engines this will be `x86_64`.